test/unit/spec/LoginRouter_spec.js

```diff
@@ -175,6 +174,17 @@ function (Okta, Q, Backbone, SharedUtil, CryptoUtil, CookieUtil, Logger, OktaAut
       expect(Logger.warn).toHaveBeenCalledWith('Field not defined in schema', arg1);
     }
 
+    var getAutoPushResponse = function (response, factorType, provider, autoPushVal) {
```

Comment:
looking at this again, do we really need the 2nd and 3rd params? They seem to be always ```'push', 'OKTA'```

Reply:
The way the backend response sends autopush is specific to a factor id. I know other factors might have push (maybe autoPush also?) at some point so i added in factorType and provider. I can remove it for now though and add those in later when the time comes?